### ISSUE TITLE
Load HPO to FHIR

### ISSUE THREAD

**Issue by chrisroederucdenver** [HOT-Ecosystem/hapi-fhir-jpaserver-starter#33]
Pyrophen and a few OWL to FHIR projects exist. We need to see what we need more than just concepts and IDs. Shahim has dealt with (is dealing with) value sets from HPO. Some spelunking to find what can be mined, or what is worth mining from elsewhere, even if just ideas, needs done.

**Comment by joeflack4** [comment]
@ShahimEssaid I think we've achieved the basic part of this. We've loaded HPO into a server. I guess this is similar to a lot of the other issues where we've essentially got an MVP of it but there's lots of fine tuning we need to do to get this in its optimal state? That's kinda where I am with Mondo on FHIR, for example.

**Comment by chrisroederucdenver** [comment]
We could stand 2 kinds of additional detail here. What is "basic part"? What more might be desired. It's an analysis task. Having some detail about the first part written down makes the second a little easier. My glance at pyrophen output is that it's basically terms/concepts, no value sets, no maps.

**Comment by joeflack4** [comment]
Yeah I'm not sure. You definitely know better than me the extent of our HPO on FHIR work. By basic part, I don't know either. I just know that something HPO related has already been completed. Just don't know what additional updates remain. But this issue would be a good place to enumerate that. I just updated your OP w/ the list of pending updates you mentioned, thanks!

**Comment by chrisroederucdenver** [comment]
OP?

Let's turn confessions of ignorance into goals to get a more clear picture. @ShahimEssaid has worked with HPO, I think to get some value sets out of it. Others may have more desires.

**Comment by joeflack4** [comment]
OP = Original Post
That's fine. I added this to the OP:
>  1. CodeSystem
>  2. ValueSet
>  3. ConceptMap
I checked off CodeSystem, but @ShahimEssaid please update the list w/ whatever is done. Maybe none of these are done yet, or are partially done.

**Comment by chrisroederucdenver** [comment]
I just got off the phone (zoom) with Tim talking about LOINC, comp-loinc and the axioms that Melissa is interested in for LOINC. This came up as a more accessible way to finally learn more OWL etc. I guess HPO, being an ontology is more consistent than LOINC and is more beginner friendly. I will make time and fire up Protege and have a look around. I can loop you in as I make sense of it. Lemme know your interest and time etc. No worries if busy.

**Comment by joeflack4** [comment]
Oh, I see. This is related to your comment (and my response) in the other HPO issue: https://github.com/HOT-Ecosystem/hapi-fhir-jpaserver-starter/issues/1#issuecomment-1262898963

I'm a little involved with CompLOINC so I have a decent grasp of what Tim's doing. I am having a really packed week though, and I think the next week or few are going to be the same, so I probably won't have time to chat about it outside of the TIMS dev meeting.
